vta/hardware/chisel/src/main/scala/core/TensorGemm.scala

```diff
@@ -61,6 +61,23 @@ class PipeAdder(aBits: Int = 8, bBits: Int = 8) extends Module {
   io.y := add
 }
 
+/** Adder */
```

Comment:
Do you mind adding a couple comments here and in Pipelined adder how the designs differ?

Reply:
explanation added